src/Renderer/c3DEngine.js

```diff
@@ -141,21 +145,68 @@ c3DEngine.prototype.getRenderer = function getRenderer() {
     return this.renderer;
 };
 
-c3DEngine.prototype.renderViewTobuffer = function renderViewTobuffer(view, buffer, x, y, width, height) {
-    // TODO Deallocate render texture
+/**
+ * Render view to a Uint8Array.
+ *
+ * @param {View} view - The view to render
+ * @param {object} [zone] - partial zone to render (zone x/y uses view coordinates)
```

Comment:
You can describe your object's property with additional `@param {number} zone.x - <desc>` tags

Reply:
Done